packages/components/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "@talend/scripts-config-typescript/tsconfig.json",
+  "include": ["src/*"],
+  "exclude": ["node_modules"],
+  "files": [],
+  "compilerOptions": {
+    "allowJs": true,
+    "checkJs": false,
```

Comment:
is there a reason to not have activated the checkJs option?

Reply:
way too much issue related to inferred types which is not enough good.